core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/toolbar/withUVEToolbar.ts

```diff
@@ -158,7 +158,7 @@ export function withUVEToolbar() {
             }),
             $infoDisplayProps: computed<InfoOptions>(() => {
                 const pageAPIResponse = store.pageAPIResponse();
-                const editorMode = store.pageParams()?.editorMode;
+                const mode = store.pageParams()?.mode;
```

Comment:
Why this is optional?

Reply:
because on init it is undefined, we need to prevent the app from crashing

Reply:
Looks we need a "first loading" state, I've been seeing this behaviour in which the UVE is blank for a second when loading, now I know why.

Reply:
Should we fix this for the stable version? Because that will probably require spiking